<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartPage</class>
 <widget class="QWidget" name="ChartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="generateCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QtSpinBox2" name="nSeriesSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nSeriesLabel">
         <property name="text">
          <string>N Series</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nPointsLabel">
         <property name="text">
          <string>N Points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QtSpinBox2" name="nPointsSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="drawLinesCheck">
       <property name="text">
        <string>Draw Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="axisOffsetCheck">
       <property name="text">
        <string>Allow Axis Offsets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fixYAxisCheck">
       <property name="text">
        <string>Fix Y Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="yMaxLabel">
         <property name="text">
          <string>Max Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QtDoubleSpinBox2" name="yMaxSpin">
         <property name="maximum">
          <double>9999999.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QtChartView" name="mainChart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtChartView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">QtChartView.h</header>
  </customwidget>
  <customwidget>
   <class>QtSpinBox2</class>
   <extends>QSpinBox</extends>
   <header location="global">QtSpinBox2.h</header>
  </customwidget>
  <customwidget>
   <class>QtDoubleSpinBox2</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">QtSpinBox2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
